GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -168,16 +164,21 @@ private void AddSubscribers()
             StillThere.CloseClassificationPanel += OnCloseClassifier;
         }
 
-        private void OnSendRequestToUser(TableUser UserToNotify)
+        public async void Load()
+        {
+            await GetWorkflow();
+            PrepareForNewClassification();
+        }
+
+        public void OnSendRequestToUser(TableUser UserToNotify)
         {
             NotificationRequest Request = new NotificationRequest(User, CurrentSubject.Id);
             Messenger.Default.Send<NotificationRequest>(Request, $"{UserToNotify.Name}_ReceivedNotification");
         }
 
-        private async void GetWorkflow()
+        public async Task GetWorkflow()
```

Comment:
Wait, does this function actually return anything?

Reply:
No, but a method marked `async` must return `Task` or `Task<T>` https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/#BKMK_ReturnTypesandParameters